src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,49 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Entity
+@Data
```

Comment:
Lombok is on a card later, since you have to add some preconditions to the setters, and with Lombok you could not do it. So wait to use it :pray: